manifests/guestbook.yaml

```diff
@@ -13,11 +12,11 @@ spec:
         app: guestbook
     spec:
       containers:
-      - image: okteto/guestbook:golang
-        name: guestbook
-        env:
-          - name: REDIS_HOST
-            value: redis
+        - image: okteto.dev/guestbook:1.0.0
```

Comment:
Don't set `image`, or keep it empty. `kustomize` will set the value

Reply:
re-added after error